src/components/AssetsPage/index.jsx

```diff
@@ -213,4 +254,9 @@ AssetsPage.propTypes = {
   searchSettings: PropTypes.shape({
     enabled: PropTypes.bool,
   }).isRequired,
+  clearAssetDeletion: PropTypes.func.isRequired,
+};
+
+AssetsPage.defaultProps = {
+  deletedAssetIndex: null,
```

Comment:
This smells funny. I get why you had to do it, but the fact that it was needed is still.. ugh.

Is there a way to force the Redux default to a number?

Reply:
The actual default in Redux is null already, so this is just mirroring that. Is that okay? Or would you prefer that the null value for this variable be a number everywhere? We could do -1 as our null value and just check that deletedAssetIndex is equal to/not equal to -1 where it matters. What do you think?

Reply:
I'm not a big fan of special-cases for null in general, but if Redux is already using `null` as a default I think that's the way to go over adding our own custom -1 logic. At least then it's clearly a "this was never set" value instead of the result of an overzealous decrement, for example.

Reply:
Sorry, I think I wasn't super clear. I'm the one that set it to `null` [here](https://github.com/edx/studio-frontend/pull/155/files#diff-1dcfcb67a72beb3a12a086472ae272aeR15). Does that change your opinion?  If I were to omit the initialization to `null`, I believe that the default value would be `undefined` instead. There is a Number object in Javascript we could leverage, perhaps (i.e. `null` might make more sense as the default for an object than for a primitive).

Reply:
Short answer: go ahead and merge this, `null` is alright here. The funny smell I referenced above is likely not something we can easily address at this time.

Longer answer: The concept of a null value in general is something I don't particularly like, for reasons that are fairly well summarized [here](https://www.lucidchart.com/techblog/2015/08/31/the-worst-mistake-of-computer-science/). Further, the fact that javascript makes it worse by having both `undefined` and `null` exist is pants-on-head dumb, similar to the famous `===` in terms of "things that should never have had reason to exist". IMO, the "cleanest" possible way to code this would be to avoid null/sentinel/hasn't-been-set-yet values at all. As a thought experiment say we split `deletedAssetIndex` into two variables `somethingWasDeletedJustNow` (boolean, never null) and `deletedAssetIndex` (number, never null). Wouldn't the propTypes be simpler to handle throughout now that we have a boolean and a number? Optionals/Maybes attempt to solve this (see https://www.npmjs.com/package/maybe as an example), but are well beyond the scope of this change.